BuildingSync.xsd

```diff
@@ -17818,6 +17819,534 @@
+                        <xs:element name="TimeSeries" type="auc:TimeSeriesType" minOccurs="0" maxOccurs="unbounded"/>
+                      </xs:sequence>
+                    </xs:complexType>
+                  </xs:element>
+                </xs:sequence>
+                <xs:attribute name="ID" type="xs:ID" use="required"/>
+              </xs:complexType>
+            </xs:element>
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+      <xs:element ref="auc:UserDefinedFields" minOccurs="0"/>
+    </xs:sequence>
+    <xs:attribute name="ID" type="xs:ID" use="required"/>
+  </xs:complexType>
+  <xs:simpleType name="IntervalFrequencyType">
+    <xs:annotation>
+      <xs:documentation>Indicates frequency of data that's available for a given variable. Data that's available can range from 1 minute interval to annual. This interval frequency can be applied to resource or other time series data like weather.</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="1 minute"/>
+      <xs:enumeration value="10 minute"/>
+      <xs:enumeration value="15 minute"/>
+      <xs:enumeration value="30 minute"/>
+      <xs:enumeration value="Hour"/>
+      <xs:enumeration value="Day"/>
+      <xs:enumeration value="Week"/>
+      <xs:enumeration value="Month"/>
+      <xs:enumeration value="Annual"/>
+      <xs:enumeration value="Quarter"/>
+      <xs:enumeration value="Other"/>
+      <xs:enumeration value="Unknown"/>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:simpleType name="UnitsType">
+    <xs:annotation>
+      <xs:documentation>Enumeration for different potential units.</xs:documentation>
+    </xs:annotation>
+    <xs:union memberTypes="auc:ResourceUnitsType auc:PressureUnitsType auc:PeakResourceUnitsType auc:TemperatureUnitsType"/>
+  </xs:simpleType>
+  <xs:simpleType name="DimensionlessUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:DimensionlessUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="PeakResourceUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:PeakResourceUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="PressureUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:PressureUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="ResourceUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:ResourceUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="TemperatureUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:TemperatureUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="OtherUnitsType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Other"/>
+      <xs:enumeration value="Unknown"/>
+      <xs:enumeration value="None"/>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:simpleType name="DimensionlessUnitsBaseType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Percent, %"/>
```

Comment:
do these align with haystack units: https://project-haystack.org/doc/Units

Reply:
![Screen Shot 2020-12-17 at 9 59 25 AM](https://user-images.githubusercontent.com/28991233/102504307-a7b1b480-404e-11eb-8e0c-9931dcd9f414.png)

Reply:
just some of them.  But yes, that is what I was trying to align to.